src/CoreLib/distributedfunction.fs

```diff
@@ -205,10 +205,27 @@ type internal SchemaBinaryFormatterHelper<'T>() =
         let o = fmt.Deserialize( ms ) 
         if Utils.IsNotNull o then o :?> 'T else Unchecked.defaultof<_>
 
+/// Helper class to generate schema ID
+type internal SchemaPrajnaFormatterHelper<'T>() = 
```

Comment:
We'd prefer not having "Prajna" in type/member name but only in namespace, especially for public visible ones. This is internal, so it's not a big deal, just point it out.